packages/babel-plugin-proposal-class-properties/src/index.js

```diff
@@ -219,21 +231,56 @@ export default function(api, options) {
           }
         }
 
+        let iife;
+
+        if (staticNodes.length > 0) {
+          // prepare iife before removing properties
+          iife = getClassIIFE(path, assumePure);
+        }
+
         for (const prop of props) {
           prop.remove();
         }
 
         if (!nodes.length) return;
 
-        if (path.isClassExpression()) {
+        if (isClassExpression) {
           path.scope.push({ id: ref });
           path.replaceWith(t.assignmentExpression("=", ref, path.node));
         } else if (!path.node.id) {
           // Anonymous class declaration
           path.node.id = ref;
         }
 
-        path.insertAfter(nodes);
+        if (staticNodes.length === 0) {
+          if (path.parentPath.isExportDeclaration()) {
+            path.parentPath.insertAfter(nodes);
+          } else {
+            path.insertAfter(nodes);
+          }
```

Comment:
You can just use `path.insertAfter(nodes)`, because of https://github.com/babel/babel/pull/7040